lib/spack/spack/fetch_strategy.py

```diff
@@ -1407,6 +1407,54 @@ def fetch(self):
             raise FailedDownloadError(self.url)
 
 
+@fetcher
+class GCSFetchStrategy(URLFetchStrategy):
+    """FetchStrategy that pulls from a GCS bucket."""
+    url_attr = 'gs'
```

Comment:
Sure, I can name it gcs. I used `gs` because the URLs are formatted as `gs://` rather than `gcs://`. But I'd happily update this `url_attr` field to match.

Reply:
So, that's actually what I was asking (sorry if that wasn't clear).

Would it make more sense to use a `gcs://` scheme for the URLs?  Or perhaps `gs://` is more "standard" among GCS users?

Reply:
Of course, I just *now* thought to google it and it does look like `gs://` is what is commonly used.  If this is right, then I think we should stick with `gs`.

Reply:
No problem at all!